src/app/shared/services/record.service.ts

```diff
@@ -41,4 +45,12 @@ export class RecordService {
       .map(res => res.json());
   }
 
+  saveRecord(record: Object): Observable<Object> {
+    let postData = {
+      'pid_type': this.type,
+      'recid': this.recid,
+      'json_data': record
+    };
+    return this.http.post('http://localhost:5000/editor/save', postData).map(res => res.json());
```

Comment:
Ok i will check it. But this router is the one that fetches record-editor in inspire view. I told about a prefix const that we will use it for rest requests. So this router is for inspire to fetch angular view of record editor. Am i right?